Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1158,13 +1158,17 @@ public BlockCommit GetBlockCommit(long index)
                 _rwlock.EnterWriteLock();
                 try
                 {
-                    if (evaluateActions && actionEvaluations is null)
+                    if (evaluateActions)
                     {
-                        _logger.Information(
-                            "Executing actions in block #{BlockIndex} {BlockHash}...",
-                            block.Index,
-                            block.Hash);
-                        ValidateBlockStateRootHash(block, out actionEvaluations);
+                        if (actionEvaluations is null)
+                        {
+                            _logger.Information(
+                                "Executing actions in block #{BlockIndex} {BlockHash}...",
+                                block.Index,
+                                block.Hash);
+                            ValidateBlockStateRootHash(block, out actionEvaluations);
+                        }
+
```

Comment:
Won't logging `Executed actions in block...` on L1176 better to be moved?
Seems a bit weired on cached situation to see those log, without `Executiong actions...` log on L1165.

Reply:
Ok, seems reasonable.